assets/js/util.js

```diff
@@ -119,6 +119,27 @@ $(document).ready(function () {
     $('.modal').on('hide.bs.modal', function () {
         $('a[data-target=#' + $(this).attr('id') + ']').parents('li').removeClass('active');
     });
+
+    $(window).scroll(function () {
+        if($(window).scrollTop() > BACK_TO_TOP_ACTIVATION_HEIGHT) {
+            $('#backToTop')
```

Comment:
@sushain97 , Um, I tried optimising the code with both of the above methods and it works fine with both! However, when I add a `hidden-xs hidden-sm` class to the `button`, and run the above code, the functionality works fine but I am unable to hide the `button` on `-md -lg` devices even with a CSS of `display: none;`. Does it mean that adding `hidden-xs hidden-sm` implicitly apply `visible-md visible-lg`?